shared/src/main/java/com/wedriveu/shared/util/Position.java

```diff
@@ -17,32 +17,58 @@
 
     public Position() { }
 
+    /**
```

Comment:
Missing docs in class definition (before `@author`), and remove `@since` annotation